CalculatorWith/CalculatorTests.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using Xunit;
+using CalculatorWithTests;
+
+
+namespace CalculatorWithTests.Tests
+{
+    public class CalculatorTests
+    {
+        [Theory]
+        [InlineData(10, 7, 17)]
+        [InlineData(2, 2, 4)]
+        [InlineData(3, 4, 7)]
+        [InlineData(-2, -2, -4)]
+        public void Add(int number1, int number2, int expectedSum)
+        { //Arrange
```

Comment:
These initial comments should probably be on the next line down instead of sharing the `{`'s line